test/ModuleInterface/Inputs/DummyFramework.framework/Modules/DummyFramework.swiftmodule/arm64.swiftinterface

```diff
@@ -0,0 +1,9 @@
+// swift-interface-format-version: 1.0
+// swift-compiler-version: Apple Swift version 5.2
+// swift-module-flags: -target arm64-apple-ios13.0 -enable-library-evolution -swift-version 5 -enforce-exclusivity=checked -O -module-name DummyFramework
+import Foundation
+import Swift
+public struct S {
+  public var x: Swift.Int
+  public init()
+}
```

Comment:
What’s the purpose of all of these declarations? Are they part of the test?

Reply:
The point is to make the swiftinterface "realistic" (and not an edge case like being empty) which still being minimal. Same for the `.swift` file (e.g. the compiler could theoretically optimize the 0 declaration case to not actually import anything perhaps?). I'll remove it if you think it is unnecessary.

Reply:
I think they unnecessarily complicate the test, mainly by obscuring what it is about the compiler that you're testing.

If you want a minimum "realistic" swiftinterface, just have `import Swift` in it—every interface other than the standard library should have that.